zmon_cli/client.py

```diff
@@ -833,7 +833,7 @@ def update_grafana_dashboard(self, grafana_dashboard: dict, **kwargs) -> dict:
 
         current_span.set_tag('grafana_dashboard_id', grafana_dashboard['dashboard']['id'])
 
-        resp = self.session.post(self.endpoint(GRAFANA), json=grafana_dashboard, timeout=self._timeout)
+        resp = self.session.post(self.endpoint(GRAFANA), json=json.dumps(grafana_dashboard), timeout=self._timeout)
```

Comment:
not clear why this is needed?

Reply:
because `grafana_dashboard` was sent as python dict and json wasn't being parsed.